source/packages/modulo_core/CMakeLists.txt

```diff
@@ -19,11 +19,13 @@ list(APPEND CMAKE_PREFIX_PATH /opt/openrobots)
 find_package(pinocchio REQUIRED)
 find_package(OsqpEigen REQUIRED)
 find_package(osqp REQUIRED)
+find_package(Protobuf REQUIRED)
```

Comment:
As discussed, `find_library(protobuf REQUIRED)` will be sufficient as we only care about the runtime `libprotobuf.so` and not any of the additional package stuff (like cmake macros, etc)